src/_base/harness/config/commands.yml

```diff
@@ -96,16 +96,20 @@ command('db console'):
 
 command('assets download'):
   env:
-    AWS_ID:  =@('aws.id')
-    AWS_KEY: =@('aws.key')
+    AWS_ACCESS_KEY_ID: = @('aws.access_key_id')
+    AWS_ID: = @('aws.access_key_id')
+    AWS_KEY: = @('aws.secret_access_key')
+    AWS_SECRET_ACCESS_KEY: = @('aws.secret_access_key')
   exec: |
     #!bash(workspace:/)|@
     passthru ws.aws s3 sync '@('assets.remote')' '@('assets.local')'
 
 command('assets upload'):
   env:
-    AWS_ID:  =@('aws.id')
-    AWS_KEY: =@('aws.key')
+    AWS_ACCESS_KEY_ID: = @('aws.access_key_id')
+    AWS_ID: = @('aws.access_key_id')
+    AWS_KEY: = @('aws.secret_access_key')
+    AWS_SECRET_ACCESS_KEY: = @('aws.secret_access_key')
   exec: |
     #!bash(workspace:/)|@
     passthru ws.aws s3 sync '@('assets.local')' '@('assets.remote')'
```

Comment:
A possible interesting sideffect of this is that prior to this, if aws was installed on host, it would use the host's ~/.aws/config (etc), and presumably now it will use the project's key